tensorflow/lite/delegates/nnapi/java/src/main/native/BUILD

```diff
@@ -18,6 +18,8 @@ cc_library(
         "notap",
     ],
     deps = [
+        "//tensorflow/lite:context",
+        "//tensorflow/lite:framework",
```

Comment:
Can we avoid pulling in the full framework here? Would like to avoid that dependency if possible.